wiki/Glossary/Break/en.md

```diff
@@ -0,0 +1,17 @@
+---
+tags:
+  - pause
+  - break Time
+---
+
+# Break
+
+*Breaks* are sections of a [beatmap](/wiki/beatmaps) which have no [hit objects](/wiki/Hit_Objects). During this time, the drain of the health bar is temporarily halted. Breaks usually allow the player to rest their arm for a short while and reposition their input device.
+
+Depending on settings, letterboxes may appear when a break is initiated. In addition to this, a passing or failing graphic and sound will appear and play to indicate if the player is doing well or not. This will only appear if there is enough time for the graphic to show.
```

Comment:
```suggestion
Depending on the enabled settings, letterboxes may appear when a break is initiated. In addition to this, a passing or failing graphic and sound may appear and to indicate if a player is doing well or not. This will only appear if there is enough time for the graphic to show.
```